.github/workflows/lint.yml

```diff
@@ -49,32 +49,6 @@ jobs:
           LINTLY_COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
 
 
-  frontend-lint:
```

Comment:
In your opinion, would it be possible to keep this linting for the dashboard folder (obviously without checking out the submodule)?

Reply:
Actually using the submodule is a little bit tricky and I don't know if it the best thing to do, I would wait on that for the moment